app/services/chat_channels/creation.rb

```diff
@@ -0,0 +1,15 @@
+module ChatChannels
+  class Creation
+    attr_accessor :user, :params
+
+    def initialize(user, params)
+      @user = user
+      @params = params
```

Comment:
same as the other. if you need `channel_name` and `slug` it's better to declare params explicitly:

`def initialize(user, channel_name:, slug:)`

this way you also get Ruby's error if the two keywords params aren't given